### PR TITLE
NG1215 - CAP cssClass setting

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will added setting for cssClass option in CAP.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-ng/issues/1215

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/contextualactionpanel/test-css-settings.html
- Click the Button
- Open `Developer Console`
- Inspect and Check to see `my-custom-panel-test` inside the CAP

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — APPROVED** [review]
Looks good - this would be a rather easy one to add a test for 👍🏻 
Also reminder to add the NG type

**Comment by tjamesallen15** [comment]
> Looks good - this would be a rather easy one to add a test for 👍🏻 Also thanks for adding the type [infor-design/enterprise-ng#1242](https://github.com/infor-design/enterprise-ng/pull/1242)

no problem @tmcconechy ! welcome!

**Comment by ericangeles** [comment]
Looks good! Agree with @tmcconechy. Once you add the test, I'll approve it. 🍌

File: app/views/components/contextualactionpanel/test-css-settings.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          fullsize: 'responsive',
+          attributes: [
+            { name: 'id', value: 'cap' },
+            { name: 'data-automation-id', value: 'cap-automation-id' }
+          ]
+        }
+    });
+  });
+</script>
+w
```
Comment (ericangeles) [inline app/views/components/contextualactionpanel/test-css-settings.html:139]: Looks like an excess character

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**